x-pack/plugins/remote_clusters/public/application/sections/components/remote_cluster_setup_trust/remote_cluster_setup_trust.tsx

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useContext } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  EuiSpacer,
+  EuiCard,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiButton,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+
+import { remoteClustersUrl } from '../../../services/documentation';
+import { AppContext } from '../../../app_context';
+import { ConfirmTrustSetupModal } from './confirm_modal';
+
+const CARD_MAX_WIDTH = 400;
+const i18nTexts = {
+  apiKeyTitle: i18n.translate(
+    'xpack.remoteClusters.clusterWizard.trustStep.setupWithApiKeys.title',
+    { defaultMessage: 'API keys' }
+  ),
+  apiKeyBadge: i18n.translate(
+    'xpack.remoteClusters.clusterWizard.trustStep.setupWithApiKeys.badge',
+    { defaultMessage: 'BETA' }
+  ),
+  apiKeyDescription: i18n.translate(
+    'xpack.remoteClusters.clusterWizard.trustStep.setupWithApiKeys.description',
+    {
+      defaultMessage:
+        'Set up your authentication mechanism with API keys to connect to your remote cluster.',
+    }
+  ),
+  certTitle: i18n.translate('xpack.remoteClusters.clusterWizard.trustStep.setupWithCert.title', {
+    defaultMessage: 'Certificates',
+  }),
+  certDescription: i18n.translate(
+    'xpack.remoteClusters.clusterWizard.trustStep.setupWithCert.description',
+    {
+      defaultMessage:
+        'This is our easier to setup method, but less secure than the API key based approach.',
+    }
+  ),
+};
+
+const docLinks = {
+  apiKey: remoteClustersUrl,
+  apiKeyCloud: remoteClustersUrl,
+  cert: remoteClustersUrl,
+  certCloud: remoteClustersUrl,
+};
```

Comment:
These docs links are just hardcoded for now to be all the same, but once the docs team finish off the doc pages we can come back to to patch this,